lib/launchers/o_c_m_cluster.rb

```diff
@@ -263,10 +263,17 @@ def create_ocpinfo_file(osd_name, dir, filename = 'OCPINFO.yml')
     def wait_for_osd(osd_name)
       loop do
         osd_status = get_value(osd_name, "state")
-        ocp_version = get_value(osd_name, "openshift_version")
+        ocp_version = "NONE"
         logger.info("Status of cluster #{osd_name} is #{osd_status} and OCP version is #{ocp_version}")
```

Comment:
Also better to relocate this logger line, too.

Reply:
Hi @liangxia I suggest to keep the logger line as I think it is useful to print actual status to the log.